config/routes.rb

```diff
@@ -0,0 +1,6 @@
+Rails.application.routes.draw do
+  devise_for :users
+  resources :posts
```

Comment:
### Routes
#### make sure you're using resource only or except tag, since you are not using all the default resources, check out [this doc](https://guides.rubyonrails.org/routing.html)